mobify/progressive-web-scaffold
# Update version of the SDK

Use v0.0.10 of the progressive-web-sdk

## How to test-drive this PR
- `npm i`
- `npm run dev`
- Preview against merlins potions and ensure that the app runs correctly
- `npm run test` make sure tests pass
